social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOC_People_Activity_CommentTestIT.java

```diff
@@ -200,6 +200,7 @@ public void test02_CommentOnYourFriendsActivity() {
     $(byId("cke_CommentTextarea" + id)).waitUntil(Condition.appears, Configuration.timeout).click();
     executeJavaScript("CKEDITOR.instances.CommentTextarea" + id + ".insertText(\"" + comment + "\")", "");
     // click on the button comment
+    Utils.pause(2000);
```

Comment:
So there is something in Selenide or in the way we use it. Did we try others ways (without using waitUntil) ?
https://github.com/codeborne/selenide/issues/424 : "P.S. I recommend to avoid using method waitUntil in most cases. You only need it if you want to set specifically special timeout for some operation. You don't need it most of the time."